crates/yas_core/src/tests/utils/constants.cairo

```diff
@@ -75,6 +75,10 @@ mod PoolConstants {
         contract_address_const::<'WALLET'>()
     }
 
+    fn OTHER() -> ContractAddress {
+        contract_address_const::<'OTHER'>()
+    }
```

Comment:
What does this constant represent?

Reply:
This is used to create contract address and the string between `''` is the key to create the test address